lock/lock.py

```diff
@@ -153,43 +149,39 @@ async def viewlock(
         `[p]viewlock #secret-channel`
         `[p]viewlock 7382395026348520 @nubs`
         """
-        try:
-            await ctx.trigger_typing()
```

Comment:
Please use `await ctx.typing()` and retain the try/except block rather than using a context manager.